src/Nethermind/Nethermind.Runner/configs/baseline.cfg

```diff
@@ -50,5 +50,14 @@
     "Enabled": false,
     "PushGatewayUrl": "http://localhost:9091/metrics",
     "IntervalSeconds": 5
+  },
```

Comment:
I wonder if I can add to this config Vault developer settings, or should I create a separate config for that?